pyvista/core/utilities/helpers.py

```diff
@@ -16,7 +16,7 @@
 
 
 def wrap(
-    dataset: Optional[Union[np.ndarray, _vtk.vtkDataSet, 'Trimesh', 'Mesh']]
+    dataset: Optional[Union[np.ndarray, _vtk.vtkDataSet, 'Trimesh', 'Mesh', list]]
```

Comment:
I think we can use `Matrix` from our `_typing_core`?

Reply:
If we allow Sequences of DataSets to be wrapped to MultiBlock then we'd also need to add that as I don't think `Matrix` includes this.

Reply:
MultiBlock support sounds appealing, but I would like to carefully consider the specifications and would prefer to do so in another PR.
```suggestion
    dataset: Optional[Union[np.ndarray, _vtk.vtkDataSet, 'Trimesh', 'Mesh', Matrix]]
```